python/python/glide/async_commands/core.py

```diff
@@ -1368,3 +1373,111 @@ async def invoke_script(
+        The callback is associated with each channel/pattern,
+        meaning subsequent subsciption for the same channel/pattern will override the previous association,
+        dereferencing the original callback object.
+        Note1: Overlapping subscriptions produced by combinations of Exact and Pattern modes
+        count as distinct subscriptions and will produce duplicated messages.
+        Note2: Patterns are not applicable in Sharded mode. That is, it is not possible to create a sharded subscription with a pattern,
+        and pattern/exact subscriptions won't receive messages which were published in Sharded mode.
+        See https://redis.io/docs/interact/pubsub for more details.
+
+        Args:
+            channels_or_patterns (Set[str]): Set of channels or patterns to subscibe to.
+            channel_mode (ChannelModes): Mode of operation. See ChannelModes.
+            callback (Callable[[CoreCommands.PubSubMsg], None]): Callback to be called for each new message.
+            For each channel or pattern there is exactly one active callback.
+                    Subsequent calls to subscribe() will reset callbacks for existing channels or patterns.
+            context (Optional[Any]): User-provided context for this subscription. Will be passed to callback as part of PubSubMsg.
+
+        Examples:
+            >>> def process_news_message(message: PubSubMsg) -> None:
+            >>>     print(f"'{message.context}' received new message '{message.message}' from channel '{message.channel}')
+            >>> await client.subscribe({"local-news"}, ChannelModes.Exact, process_news_message, "example-client")
+                1  # Subscribes to local-news channel with process_news_message callback function
+            >>> def process_all_messages(message: PubSubMsg) -> None:
+            >>>     print(f"'{message.context}' received new message '{message.message}' from channel '{message.channel}' by pattern '{message.pattern}')
+            >>> await client.subscribe({"*"}, ChannelModes.Pattern, process_all_messages, "example-client")
+                2  # Subscribes to all channels with process_all_message callback function
+        """
+        ...
+
+    async def publish(self, message: str, channel: str, sharded: bool = False) -> int:
+        """
+        Publish message on channels.
+        See https://redis.io/docs/interact/pubsub for more details.
+
+        Args:
+            message (str): Message to publish.
+            channel (str): Channel to publish the message on.
+            sharded (bool): Use sharded PUBSUB mode.
+
+        Returns:
+            int: Number of clients that received the message.
+
+        Examples:
+            >>> await client.publish("Hi all!", "global-channel", True)
+                1  # Publishes "Hi all!" message on global-channel channel using sharded mode
+        """
+        ...
+
+    async def unsubscribe(
```

Comment:
We are adding the redis documentation for all commands, I see you added only a link to https://redis.io/docs/interact/pubsub. We should have a like to https://redis.io/commands/unsubscribe/ (or other commands) as well